report/assets/styles.css

```diff
@@ -1435,9 +1437,40 @@
   margin: 0px auto;
 }
 
-.lh-category-header .lh-audit__title {
-  font-size: var(--category-header-font-size);
-  line-height: var(--header-line-height);
+.lh-category-header__finalss {
+  display: grid;
+  grid-template-columns: 1fr 1px 1fr;
+  justify-items: center;
+  align-items: center;
+  gap: var(--report-line-height);
+  margin-bottom: var(--default-padding);
```

Comment:
this provides the same 12px as the rule it overrides. remove?

Reply:
ah good catch. i spotted it was a little cramped and thought this should fix it... but.. yeah i didnt realize it's a noop. 

we may want to double this instead.. will also depend on the vishidden/pill stuff.  

tbd.